### ISSUE TITLE
Bug when applying RandomHorizontalFlip()

### ISSUE THREAD

**Issue by JiangPQ** [pytorch/accimage#11]
This is what I get when I applying RandomHorizontalFlip():
![bug](https://user-images.githubusercontent.com/10220802/42192845-feba7930-7e9d-11e8-9227-8bdf1d4ec78b.png)
As you can see **the upper part of some images is totally disrupted**.
Here is how I use the accimage:
`
    import accimage

    def get_img_loader(batch_size, shuffle=True, num_workers=8, mode='train', img_size=227):
        t_dir = os.path.join(imagenet_ssd_dir, 'train')
        v_dir = os.path.join(imagenet_ssd_dir, 'val')

        transform_t = Compose([
            RandomResizedCrop(img_size, (1.0, 1.0), (1.0, 1.0)),
            RandomHorizontalFlip(),
            ToTensor(),
            # Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
        ])
        transform_v = Compose([
            CenterCrop(img_size),
            ToTensor(),
            Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
        ])
        if mode == 'val':
            data_folder = ImageFolder(v_dir, transform_v, loader=accimage.Image)
        else:
            data_folder = ImageFolder(t_dir, transform_t, loader=accimage.Image)

        return DataLoader(data_folder, batch_size=batch_size,shuffle=shuffle, num_workers=num_workers)

`

**Comment by Maratyszcza** [comment]
Have you tried to update Intel IPP library? The code in `accimage` is very simple, and looks correct: https://github.com/pytorch/accimage/blob/master/imageops.c#L160-L167

**Comment by willprice** [comment]
I think I've fixed this in this commit: https://github.com/pytorch/accimage/pull/14/commits/d9b0bd08a476cc1fb59c1ec83aef0b6664bd9562

**Comment by willprice** [comment]
Fixed in #ea4d04d227270314033ccba4747db72fca28a223
